cypress/integration/step_definitions/Ui/doubleVanities.js

```diff
@@ -10,6 +10,11 @@ When('User clicks to Double Bathroom Vanities', () => {
     HomePage.clickDoubleVanities();
 });
 
-
+When('I test all links on the page', () => {
+  cy.checkBrokenLinks();
+});
```

Comment:
line 14 and line 17 calls the same function. If two steps are doing same thing why duplicate code?

Reply:
but you have already verified in one of the steps :)